modules/Behaviour/behaviour_manage_editProcess.php

```diff
@@ -76,7 +77,7 @@
                 $level = $_POST['level'] ?? null;
                 $comment = $_POST['comment'] ?? '';
                 $followup = $_POST['followup'] ?? '';
-                $gibbonPlannerEntryID = !empty($_POST['gibbonPlannerEntryID']) ? $_POST['gibbonPlannerEntryID'] : null;
+                $gibbonPlannerEntryID = $_POST['gibbonPlannerEntryID'] ?? null;
```

Comment:
Certain areas of the codebase need to use a !empty() to properly handle values where an empty string needs to be stored as null in the database. In this case, this variable didn't need changed into a null coalesce, it was already being handled by a ternary. Can you change this one back? Chances are, if you see a ternary with a `!empty()`, it may be there for a good reason.